src/components/campaignLandingPage/ConnectWithUs.tsx

```diff
@@ -1,133 +1,103 @@
-              Get updates from{' '}
-              {placeholders.fieldConnectWithUs.entity.fieldExternalLink.title}
-            </h2>
-          </div>
+    <div
+      className="vads-l-grid-container vads-u-padding-y--3 vads-u-padding-x--4 desktop-lg:vads-u-padding-x--0"
+      data-testid="connect-with-us"
+    >
+      <div className="vads-l-row">
+        <div className="vads-l-col--12 medium-screen:vads-l-col--9">
+          <p className="va-u-text-transform--uppercase vads-u-color--gray-medium vads-u-font-size--sm vads-u-margin-bottom--0">
+            Connect with us
+          </p>
+          <h2 className="vads-u-margin-top--0">
+            Get updates from {organizationTitle}
+          </h2>
         </div>
+      </div>
 
-        <div className="vads-l-row medium-screen:vads-u-margin-x--neg1">
-          {placeholders.fieldConnectWithUs.entity.fieldEmailUpdatesLink.url
-            .path &&
-            placeholders.fieldConnectWithUs.entity.fieldEmailUpdatesLink
-              .title && (
-              <div className="vads-l-col--12 medium-screen:vads-l-col--4">
-                <div className="vads-u-margin-y--1 medium-screen:vads-u-margin-x--1">
-                  <va-icon
-                    icon="mail"
-                    size="3"
-                    className="vads-u-color--link-default vads-u-padding-right--1"
-                  ></va-icon>
-                  <va-link
-                    href={
-                      placeholders.fieldConnectWithUs.entity
-                        .fieldEmailUpdatesLink.url.path
-                    }
-                    text={
-                      placeholders.fieldConnectWithUs.entity
-                        .fieldEmailUpdatesLink.title
-                    }
-                  />
-                </div>
-              </div>
-            )}
+      <div className="vads-l-row medium-screen:vads-u-margin-x--neg1">
+        {emailLink?.href && emailLink?.title && (
+          <SocialMediaLink
+            icon="mail"
+            href={emailLink.href}
+            text={emailLink.title}
+          />
+        )}
 
-          {socialLinksObject.twitter?.value && (
-            <div className="vads-l-col--12 medium-screen:vads-l-col--4">
-              <div className="vads-u-margin-y--1 medium-screen:vads-u-margin-x--1">
-                <va-icon
-                  icon="x"
-                  size="3"
-                  className="vads-u-color--link-default vads-u-padding-right--1"
-                ></va-icon>
-                <va-link
-                  href={`https://twitter.com/${socialLinksObject.twitter.value}`}
-                  text={`${placeholders.fieldConnectWithUs.entity.fieldExternalLink.title} X (formerly Twitter)`}
-                />
-              </div>
-            </div>
-          )}
+        {socialLinks.twitter && (
+          <SocialMediaLink
+            icon="x"
+            href={`https://twitter.com/${socialLinks.twitter}`}
```

Comment:
Should this go straight to `x.com` now?

Reply:
In theory I would think so 🤔 That's probably worth it's own ticket, since there are other places we're using twitter.com and it probably muddies the waters to deviate from the drupal template in this PR.